### DISCUSSION TITLE
Allow # of guests graph to go above 5000

### DISCUSSION THREAD

**Discussion by MarcelVos96** [OpenRCT2/OpenRCT2#15961] [Feature requests] [answered]
Currently the graph for the amount of guests in your park only goes above 5000. It would be neat, especially with the new limits, if it could higher than 5k if you get that many guests in your park.
![image](https://user-images.githubusercontent.com/25930360/142892596-3094d7b1-ef4d-4433-8873-d88a612835ae.png)

**Comment by ZehMatt** [comment]
We should maybe look into dynamically scaling the graph, if we have it right at 65k it would look quite tiny when you start.

**Comment by ZehMatt** [comment]
There is also #15350 which would solve this

**Comment by J-PIE-314** [comment, marked as answer]
This has been implemented at some point in development.
